data_classes_test.py

```diff
@@ -0,0 +1,51 @@
+"""
+Unit tests for dataset.py
+"""
+
+import unittest
+from datetime import datetime
+from thousandaire.dataset import Data
+
+class TestData(unittest.TestCase):
+    """
+    Unit test object for Data.
+    """
+    def __init__(self, *args, **kargs):
+        unittest.TestCase.__init__(self, *args, **kargs)
+        self.data = None
+
+    def setUp(self):
```

Comment:
what is this for?

Reply:
This the method in unit test. 
It would help to reset data into the original condition before carried out each test method.

Reply:
Interesting. Do we still need line 15 given using this one?

Reply:
Oh! It truly do not need line 15.